### ISSUE TITLE
How to update my Shadowsocks-libev?

### ISSUE THREAD

**Issue by gulaholic** [shadowsocks/shadowsocks-libev#1545]
Hello,

How can I update my Shadowsocks-libev version?
I'm on Debian 7 and I can't find any shadowsocks repo related to Debian 7.

I installed the shadowsocks using apt-get.

Thanks.

### What version of shadowsocks-libev are you using?
2.4.0-1

### What operating system are you using?
Debian 7 - Wheezy

### What did you do?
Try to update the latest version

### What did you expect to see?
Version updated

### What did you see instead?
Nothing

**Comment by ragnaroks** [comment]
try to compile source code,it's always work fine

**Comment by gulaholic** [comment]
Hi,

So I need to remove the old installation first then re compile the source code?

**Comment by terrytw** [comment]
No, you can compile first, and uninstall the old version after you successfully get the installation file of new version.

**Comment by gulaholic** [comment]
Hi,

Thank you dor your recommendation, I'll consider that.

I'm afraid upgrading to Debian 8 will break some programs.

I put Squid, OpenVPN, Softether and others beside Shadowsocks.

**Comment by gulaholic** [comment]
One question, if I managed to upgrade sucessfully  to Debian 8 or 9, can I update Shadowsocks libev using apt-get? What should I do so I can do that?

**Comment by terrytw** [comment]
I am afraid that you cannot apt-get the latest version of shadowsocks even
if you upgrade to debian 8 or 9, they have 2.63 in the package source,
while the latest version is 3.06. However, as you know, debian 9 is just
released, a lot of work is still underway, and rogers0, the developer who
puts shadowsocks in debian package source, is very active, I believe you
will be able to apt-get the latest shadowsocks on debian 9 within the next
few months.

2017年6月29日 13:28，"gulaholic" <notifications@github.com>写道：
